File: JAVA/Daily Flash Codes/Week2/Day3_Jan22/code/prog2.java
import java.io.*;

class Program {
	public static void main(String args[]) throws IOException{

		BufferedReader br = new BufferedReader (new InputStreamReader(System.in));
		System.out.println("Enter two nums: ");
		int num1 = Integer.parseInt(br.readLine());
		int num2 = Integer.parseInt(br.readLine());
		
		System.out.println("Multiplication : "+ (num1 * num2));

		if(num1 > num2)	
			System.out.println("Division: "+ (num1 / num2));
		else
			System.out.println("Division : "+ (num2 / num1));
		

	}
}

File: C++/Daily Flash/Week2/Day3_Jan22/code/prog2.cpp
#include<iostream>

int main() {

	int num1,num2;
	std::cout << "Enter number 1: " ;
	std::cin >> num1;

	std::cout << "Enter number 2: " ;
	std::cin >> num2;


	
	//multiplication
	std::cout << "Mulitplication: " << num1 * num2 << std::endl;	

	//division
	if(num1 > num2)
		std::cout << "Division: " << num1 / num2 << std::endl;
	else
		std::cout << "Division: " << num1 / num2 << std::endl;
        return 0;
        
 }


